util/generate-releases.js

```diff
@@ -0,0 +1,423 @@
+  };
+
+  const downloadURL = new URL(getValue('downloadURL'));
+  const currentVersion = getValue('latestVersion');
+
+  const dirs = downloadURL.pathname.split('/');
+  dirs.shift();
+
+  if (
+    downloadURL.hostname !== 'github.com' ||
+    dirs[2] !== 'releases' ||
+    currentVersion === '最新'
+  ) {
+    console.error(red('The package is not on GitHub.'));
+    return;
+  }
+
+  let releasesAvailable = false;
+  try {
+    const res = await octokit.rest.repos.listReleases({
+      owner: dirs[0],
+      repo: dirs[1],
+    });
+
+    const generateRelease = async (release) => {
+      const result = {
+        newReleaseObj: undefined,
+        message: [],
+      };
+      try {
+        let tag = release.tag_name;
+
+        // only for hebiiro's packages
+        if (dirs[0] === 'hebiiro') {
+          const versionArray = tag
+            .split('.')
+            .filter((value) => /[0-9]+/.test(value));
+          if (versionArray.length >= 1) {
+            tag = versionArray.join('.');
+          } else {
+            throw new Error('A version-like string is not found.');
+          }
+        }
+
+        result.message.push(whiteBright(`[${tag}]`));
+
+        // Create integrity
+        const oldReleaseObj = packageItem
+          .find((value) => 'releases' in value)
+          ?.releases?.at(-1);
+        if (!oldReleaseObj?.release) {
+          throw new Error('The old release data does not exist.');
+        }
+        if (
+          packageItem
+            .find((value) => 'releases' in value)
+            ?.releases.some((p) => p[':@'].$version === tag)
+        )
+          throw new Error('The release data of the same version exist.');
+
+        const assets = release.assets.filter(
+          (asset) => extname(asset.name) === '.zip'
+        );
+        let archiveData = {
+          name: dirs[0] + '-' + dirs[1] + '-' + tag + '.zip',
+          browser_download_url: `https://github.com/${dirs[0]}/${dirs[1]}/archive/refs/tags/${res.data.tag_name}.zip`,
+        };
+        if (assets.length === 1) {
+          archiveData = assets[0];
+        } else if (assets.length >= 2) {
+          const temp = assets.find((value) => {
+            const matchPattern = new RegExp(
+              '^' + archiveNamePattern[id] + '\\.zip$'
+            );
+            return matchPattern.test(value.name);
+          });
+          if (temp) {
+            archiveData = temp;
+          } else {
+            throw new Error(
+              'No assets are found.\n  ID: ' +
+                id +
+                '\n  Pattern: ' +
+                archiveNamePattern[id]
+            );
+          }
+        } else {
+          result.message.push(
+            yellow(
+              'There seem to be no assets. So use an archive file of the source code.'
+            )
+          );
+        }
+
+        const url = archiveData.browser_download_url;
+
+        const archivePath = resolve('util/temp/archive', archiveData.name);
```

Comment:
```suggestion
        const archivePath = resolve('util/temp/' + tag + '/archive', archiveData.name);
```
To prevent the generation of files with the same name.